contracts/aggregator-proxy/src/contract.rs

```diff
@@ -324,10 +325,21 @@ mod tests {
 
     pub fn instantiate_link(app: &mut App) -> Addr {
         let link_id = app.store_code(contract_link_token());
+        let main_balance = Cw20Coin {
+            address: Addr::unchecked("owner").into(),
+            amount: Uint128::from(1_000_000_000 as u128),
```

Comment:
Is this value supposed to be hardcoded here? Wouldn't that result in minting `1_000_000_000` at the time of instantiation?

Reply:
does it matter in the tests?
With the real instantiation we pass that in the InstantiateMsg

Reply:
Oops, my bad. :D I did not notice this was part of the tests.